components/src/core/selection-toolbar/selection-toolbar.component.scss

```diff
@@ -0,0 +1,40 @@
+.pxb-selection-toolbar-icon-wrapper {
+    display: flex;
+    align-items: center;
+    z-index: 1;
+    margin-right: 24px;
+    &.pxb-selection-toolbar-no-icon {
+        margin-right: 0;
+    }
+}
+
+.pxb-selection-toolbar-title-container {
+    display: flex;
+    padding-right: 16px;
+    justify-content: center;
+    align-self: stretch;
+    flex-direction: column;
+    width: calc(100% - 56px);
+    box-sizing: border-box;
+    z-index: 1;
+
+    .pxb-selection-toolbar-title {
+        line-height: 1.6rem;
+        overflow: hidden;
+        text-overflow: ellipsis;
+    }
+
+    .pxb-selection-toolbar-subtitle-container {
+        margin-top: -2px;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        display: flex;
+        align-items: center;
+        width: fit-content;
```

Comment:
what is this doing?

Reply:
The subtitle click area was taking up the full width of the toolbar so the dropdown click event could be triggered by clicking anywhere in that area. Fit-content makes sure the click area is only as big as the subtitle. If you know a cleaner way to handle this, I'm all ears.

Reply:
Using display: block from the above comment and not flexing the title content in the toolbar seems to eliminate the need for this.